src/utils/chart.js

```diff
@@ -335,17 +335,17 @@ export const pruneIncompleteStackedAreaInterval = (
 ) => {
   const { from: dateFrom, to: dateTo } = dateRange;
 
+  const dataClone = cloneDeep(data);
```

Comment:
what kind of issues were you seeing here, and I think you may not need this

This is doing the exact same thing at L342. 

cloneDeep in our utils does the same thing too:
https://github.com/cfpb/ccdb5-ui/blob/main/src/utils/index.js#L209

Reply:
Would this work without adding this?  This looks like it is doing the same thing twice

I would change L342 to 
let filteredData = cloneDeep(data);